src/Helper/AssetsHelper.php

```diff
@@ -77,9 +82,21 @@ public function embed ($value) : string
 			else
 			{
 				$storedAsset = $this->assetsManager->getAssetMap()->get($value);
-				$fileType = $this->fileTypeRegistry->getFileType($storedAsset->getAsset());
-				$path = $this->buildUrl($storedAsset);
-				$returnValue .= $fileType->getEmbedCode($path);
+
+				if (null !== $storedAsset)
+				{
+					$fileType = $this->fileTypeRegistry->getFileType($storedAsset->getAsset());
+
+					if ($asset->getExtension() === "svg")
+					{
+						$returnValue .= $this->fileLoader->loadFile($this->assetsManager->getAssetMap(), $asset, $this->kernel->isDebug());
```

Comment:
Not sure if this is the right fix. Because embed should only work for `svg`s in the first place, and then it should just use the `FileLoader` and load the file untouched